orderbook/usecase/orderbook_usecase_test.go

```diff
@@ -549,7 +551,7 @@ func (s *OrderbookUsecaseTestSuite) TestGetActiveOrders() {
 				poolsUsecase.GetAllCanonicalOrderbookPoolIDsFunc = s.GetAllCanonicalOrderbookPoolIDsFunc(
 					nil,
 					s.NewCanonicalOrderBooksResult(1, "A"),
-					s.NewCanonicalOrderBooksResult(1, "B"),
+					s.NewCanonicalOrderBooksResult(2, "B"),
```

Comment:
Pre-existing error, started causing issue post-cache since cache is keyed by pool ID